python/ray/llm/_internal/batch/utils.py

```diff
@@ -60,3 +63,53 @@ def __len__(self):
 
     tokenizer.__class__ = CachedTokenizer
     return tokenizer
+
+
+def maybe_pull_model_tokenizer_from_s3(model_source: str) -> str:
```

Comment:
Same comment of unification between serve and data happens here. We have CloudModelDownloader which already implements `get_model(tokenizer_only=True/False)`. It's well tested and everything. We should use that and avoid creating new stuff here.